pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1500,7 +1501,9 @@ func getBackupDetailAndManifest(
 		// This is complex right now, but should be easier shortly.
 		// TODO(benbardin): Support verifying actual existence of localities for
 		// each layer after deprecating TO-syntax in 22.2
-		if !setsAreEqual(localityKVs, prevLocalityKVs) {
+		sort.Strings(localityKVs)
+		sort.Strings(prevLocalityKVs)
+		if !reflect.DeepEqual(localityKVs, prevLocalityKVs) {
```

Comment:
whoops, this was supposed to submit with that approval, sorry:

`reflect.DeepEqual()` returns false if one slice is nil, and the other is just empty, so I think you need to check either both == len(0) or deepequal.

Reply:
done.